From a9451055464272af863879dd5e5721d9860c2eb0 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Wed, 6 Sep 2023 14:01:06 +0200
Subject: [PATCH] Fix check for msgpack

Msgpack renamed it's pkg-config file, so we need to check for both
names.
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index dc864a7..d35308f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -17,6 +17,7 @@ AM_PROG_AR
 LT_INIT
 
 # Checks for libraries.
+PKG_CHECK_MODULES([MSGPACK], [msgpack-c],,[PKG_CHECK_MODULES([MSGPACK], [msgpack])])
 PKG_CHECK_MODULES([MSGPACK],[msgpack])
 PKG_CHECK_MODULES([CZMQ],[libczmq])
 
-- 
2.41.0

